politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -597,6 +637,43 @@ func (c *ctx) inventory() error {
 		return nil
 	}
 
+	// Get records metdata and store proposal names to print them. If number of
+	// tokens exceeds the Records route page size, fetch records page by page.
+	//
+	// Store proposal name is a map[token] => name.
+	names := make(map[string]string, len(tokens))
+	remainingTokens := tokens
+	for {
```

Comment:
It's better to put the break condition in the for loop statement.  It reduces the chance that an infinite loop bug is introduced at some point in the future.

Reply:
Totally agree!